src/elements/select/Select.js

```diff
@@ -6,16 +6,26 @@ import { KeyCodes } from './../../utils/key-codes/KeyCodes';
 @Component({
     selector: 'novo-select',
     directives: [COMMON_DIRECTIVES],
-    inputs: ['options', 'placeholder', 'readonly'],
+    inputs: ['options', 'placeholder', 'readonly', 'footerConfig'],
     outputs: ['onSelect'],
     template: `
         <button (click)="toggleActive($event)" tabIndex="-1" type="button" [ngClass]="{empty: empty}">{{selected.label}}<i class="bhi-collapse"></i></button>
-        <ul class="novo-select-list" tabIndex="-1">
+        <ul class="novo-select-list" tabIndex="-1" [ngClass]="{footer: footerConfig}">
             <ng-content></ng-content>
             <li *ngFor="let option of options; let i = index" [ngClass]="{active: option.active}" (click)="onClickOption(option, i)" [attr.data-automation-value]="option.label">
               <span>{{option.label}}</span>
               <i *ngIf="option.active" class="bhi-check"></i>
             </li>
+            <div *ngIf="footerConfig" class="select-footer">
+                <button  *ngIf="!footer.open" (click)="toggleFooter($event); false" tabIndex="-1" type="button" class="footer"><i class="bhi-add-thin"></i>{{footerConfig.label}}</button>
+                <div *ngIf="footer.open" [ngClass]="{active: footer.open}">
+                    <input autofocus type="text" [placeholder]="footerConfig.placeholder" [attr.id]="name" autocomplete="false" [(ngModel)]="footer.value" [ngClass]="{invalid: !footer.valid}"/>
+                    <footer>
+                        <button (click)="toggleFooter($event, false)">Cancel</button>
```

Comment:
The "Save/Cancel" should be added to the https://github.com/bullhorn/novo-elements/blob/master/src/services/novo-label-service.js, then you can inject that into Select and use `{{labels.save}}` instead of the plain text. This is used to localize it else where.